src/Readers/Vipr.Reader.OData.v4/Extensions.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Vipr.Core.CodeModel;
+using Vipr.Core.CodeModel.Vocabularies.Capabilities;
+using Vipr.Reader.OData.v4.Capabilities;
+
+namespace Vipr.Reader.OData.v4
+{
+    public static class Extensions
+    {
+#if false
```

Comment:
This file should just be removed since it effectively is doing nothing, unless you think this code will be re-enabled at some point in the future.